types/src/aggregator.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::state_store::{state_key::StateKey, table::TableHandle};
+use move_core_types::account_address::AccountAddress;
+
+#[derive(Debug, Clone, Copy, Hash, Eq, PartialEq, Ord, PartialOrd)]
+pub struct AggregatorHandle(pub AccountAddress);
+
+/// Uniquely identifies each aggregator instance in storage.
+#[derive(Debug, Clone, Hash, Eq, PartialEq, Ord, PartialOrd)]
+pub struct AggregatorID(StateKey);
+
+impl AggregatorID {
```

Comment:
hmm, where do you need AggregatorID outside?

Reply:
asking as this ID is getting deprecated and refactored, and shouldn't be needed outside

Reply:
This is needed for the `BlockExecutorTransaction` interface.